lib/_tls_common.js

```diff
@@ -56,6 +56,15 @@ function SecureContext(secureProtocol, secureOptions, context) {
   if (secureOptions) this.context.setOptions(secureOptions);
 }
 
+SecureContext.prototype.getTicketKeys = function getTicketKeys(keys) {
+  return this.context.getTicketKeys(keys);
```

Comment:
I realize you copied this from elsewhere but this method doesn't take any arguments, it returns a new buffer.

Reply:
@bnoordhuis quite right, I actually took my cues from https://github.com/ryzokuken/node/blob/6269853a9a444a3fbed351ddeb12a6e97114543d/lib/_tls_wrap.js#L928

Reply:
I'd look into the original context's `getTicketKeys()` method and try working with that one.

Reply:
Umm, it _can_ take arguments? Idk, the code was:

```cc
void SecureContext::GetTicketKeys(const FunctionCallbackInfo<Value>& args) {
#if !defined(OPENSSL_NO_TLSEXT) && defined(SSL_CTX_get_tlsext_ticket_keys)

  SecureContext* wrap;
  ASSIGN_OR_RETURN_UNWRAP(&wrap, args.Holder());

  Local<Object> buff = Buffer::New(wrap->env(), 48).ToLocalChecked();
  memcpy(Buffer::Data(buff), wrap->ticket_key_name_, 16);
  memcpy(Buffer::Data(buff) + 16, wrap->ticket_key_hmac_, 16);
  memcpy(Buffer::Data(buff) + 32, wrap->ticket_key_aes_, 16);

  args.GetReturnValue().Set(buff);
#endif  // !def(OPENSSL_NO_TLSEXT) && def(SSL_CTX_get_tlsext_ticket_keys)
}
```

Reply:
Anyway, I think most if not all people call it without any args plus no arguments mentioned in the docs, so I'll make the change.

Reply:
You mean `args`?  Any C++ API function takes that as its argument.

If you're referring to `args.Holder()`, that's the `this` object.  Positional arguments are accessed using array notation, i.e., `args[0]`.

Reply:
@bnoordhuis TIL, I guess 😅 Not super good at the C++ part of things.

Reply:
In that case, `getTicketKeys` indeed doesn't use any arguments while `setTicketKeys` takes one, perfect.